examples/docs/01_Intro.ipynb

```diff
@@ -148,7 +148,9 @@
    "metadata": {},
    "source": [
     "The ZnTrack class generates an `__init__` method for all `zn.params` and other inputs automatically.\n",
-    "When writing a custom `__init__` it is important to add `super().__init__(**kwargs)` for ZnTrack to work.\n",
+    "The generated `__init__()` code will call a method named `_post_init_()` if it exists.\n",
+    "You can use override this method to add custom initialization code.\n",
+    "Alternatively, when writing a custom `__init__` it is important to add `super().__init__(**kwargs)` for ZnTrack to work.\n",
```

Comment:
perphaps i'm not familiar enough with data classes but it is not clear what the advantage is of writing a custom `_init_` over a `_post_init_`.